controllers/utils/validate.ts

```diff
@@ -0,0 +1,90 @@
+import emailValidator from "email-validator";
+import type { RequestHandler } from "express";
+
+export const email = (email: string): boolean => {
+    return emailValidator.validate(email);
+};
+
+interface returnObj {
+    objHasProps: boolean;
+    missing?: string;
+    blank?: string;
+}
+
+export const checkObjHasProps = (
+    reqBody: object,
+    requiredProps: string[]
+): returnObj => {
+    let objHasProps: boolean = true;
+    let missing: string[] = [];
+    let blank: string[] = [];
```

Comment:
hmm what is blank?

Reply:
the props. It's defined (not missing) but has a blank value. Will change the property name to communicate that :)